File: bot.js
const TeleBot = require('telebot');
const bot = new TeleBot('466035657:AAEvvOBlKMdcxqg7IDnngnHC2vRbc2qAkbM');

//bot.on('text', (msg) => msg.reply.text(msg.text));
bot.on('sticker', (msg) => {
    return msg.reply.sticker("You just sent a sticker, wow!", { asReply: true });
});
bot.on(['/start', '/hello'], (msg) => msg.reply.text('Welcome!'));

bot.start();

File: bot.py
import config
import telebot
import os
import time
from telebot import types
import game_play

bot = telebot.TeleBot(config.token)

#starting
@bot.message_handler(commands=["start"])
def start_conversation(message):
	bot.send_message(message.chat.id, "Hello, " + message.from_user.first_name + "!")

#choosing something
@bot.message_handler(commands=["choose"])
def choose_something(message):
	markup = types.ReplyKeyboardMarkup(one_time_keyboard=True)
	markup.row('1', '2', '3')
	bot.send_message(message.chat.id, "What number do you want to choose?", reply_markup=markup)

#returning result
@bot.message_handler(func=lambda message: True, content_types=['text'])
def return_answer(message):
	text_result = game_play.check_number_and_return_result(int(message.text))
	bot.send_message(message.chat.id, text_result)

if __name__ == '__main__':
    bot.polling(none_stop=True)